wicket/src/state/update.rs

```diff
@@ -105,3 +104,15 @@ impl RackUpdateState {
         }
     }
 }
+
+#[allow(unused)]
+pub fn artifact_title(kind: KnownArtifactKind) -> &'static str {
+    type K = KnownArtifactKind;
+    match kind {
```

Comment:
I'd just write `use KnownArtifactKind::*` here personally :)

Reply:
Damnit. That is better. haha